packages/react/src/components/Modal/Modal.base.tsx

```diff
@@ -468,6 +469,7 @@ export const ModalBase: React.FunctionComponent<IModalProps> = React.forwardRef<
       (isModalOpen && modalResponsiveMode! >= (responsiveMode || ResponsiveMode.small) && (
         <Layer ref={mergedRef} {...mergedLayerProps}>
           <Popup
+            {...popupProps}
```

Comment:
I'd suggest spreading this at the bottom, so it allows overriding the role, label, description, etc.

Reply:
Updated :) Thanks @smhigley !